tools/ci_build/github/azure-pipelines/android-arm64-v8a-QNN-crosscompile-ci-pipeline.yml

```diff
@@ -32,7 +32,7 @@ parameters:
 - name: QnnSdk
   displayName: QNN SDK version
   type: string
-  default: 2.27.0.240926
+  default: 2.28.0.241029
```

Comment:
FYI about https://github.com/microsoft/onnxruntime/pull/22434 which is adding another place where the version is specified.